### PR TITLE
Added BTS support

### LINKED ISSUE #143: Add support for BTS addresses in encoding library
Here is the encoding library: https://github.com/ensdomains/address-encoder

Example: https://github.com/ensdomains/address-encoder/pull/47/files

Discord for dev help: https://discord.com/invite/AskZbFx

Please provide a reference link so we can check you did the encoding correctly.

We'd like to keep the encoding library as small as possible, so please be efficient in your use of space.

We will likely be reviewing PRs and paying out bounties more or less on a weekly basis. No need to ping us to check your PR, we will definitely get to it.

Thanks for your help!

### PR DESCRIPTION
**Pull request by canokaue**
Regarding bounty:
https://gitcoin.co/issue/ensdomains/address-encoder/143/100023845

## Issue number

Closes #143

## Description

Add BitShares support.

## List of features added/changed

Add BTS on:
src/index.ts
src/tests/index.test.ts
README.md
Already sorted by Cointype.

NOTE: If you are adding new coin address support, please make sure to add a reference link to README so that reviewer can verify.
https://github.com/satoshilabs/slips/blob/master/slip-0044.md
https://dev.bitshares.works/en/master/bts_guide/index_faq.html#dev-faq17

## How Has This Been Tested?

Local test run
TravisCI

## Checklist:

- [x] My code follows the code style of this project.
- [x] My code implements all the required features.

### CODE REVIEW

**Comment by makoto** [comment]
Please resolve conflicts (I can merge this in once resolved).

**Comment by canokaue** [comment]
@makoto conflicts resolved!

File: src/index.ts [hunk 1]
```diff
@@ -579,6 +579,40 @@ function steemAddressDecoder(data: string): Buffer {
   return Buffer.from(key);
 }
 
+// Referenced from following
+// https://dev.bitshares.works/en/master/bts_guide/index_faq.html#dev-faq17
+function btsAddressEncoder(data: Buffer): string {  
+  const RIPEMD160 = require('ripemd160');
```
Comment (makoto) [inline src/index.ts:585]: Please use `import` at the beginning of the file rather than requiring on the spot

Comment (canokaue) [reply src/index.ts:585]: Sure, on it

**Comment by canokaue** [comment]
@makoto as requested, corrected requires from BTS functions. Did the same for STEEM since I was at it.

**Pull request merged**